### PR TITLE
C9 edits

### PR DESCRIPTION
**Pull request by Lvulis**
This addresses some pieces of #837 but isn't comprehensive. Note **raster** instead of **terra** is still imported because of apparent **mapview** issues.

### CODE REVIEW

File: 09-mapping.Rmd [hunk 1]
```diff
@@ -460,6 +460,7 @@ In the second step, we create a base map showing the New Zealand's Southern Alps
 This is a place where the most important message is stated. 
 
 ```{r 08-mapping-17}
+nz_elev = rast(system.file("raster/nz_elev.tif", package = "spDataLarge"))
```
Comment (Robinlovelace) [inline 09-mapping.Rmd:463]: :+1: but not sure if this is already covered in the v4 branch.

File: 09-mapping.Rmd [hunk 2]
```diff
@@ -676,7 +677,7 @@ trails |>
   st_intersection(franconia[franconia$district == "Oberfranken", ]) |>
   st_collection_extract("LINE") |>
   mapview(color = "red", lwd = 3, layer.name = "trails") +
-  mapview(franconia, zcol = "district", burst = TRUE) +
+  mapview(franconia, zcol = "district", burst = FALSE) +
```
Comment (Robinlovelace) [inline 09-mapping.Rmd:680]: Not sure about this. Is this correct usage @tim-salabim?

File: 09-mapping.Rmd [hunk 3]
```diff
@@ -934,7 +935,7 @@ This behavior is illustrated in the subsequent code chunk which generates Figure
 ```{r nz-plot, message=FALSE, fig.cap="Map of New Zealand created with plot(). The legend to the right refers to elevation (1000 m above sea level).", fig.scap="Map of New Zealand created with plot()."}
 g = st_graticule(nz, lon = c(170, 175), lat = c(-45, -40, -35))
 plot(nz_water, graticule = g, axes = TRUE, col = "blue")
-raster::plot(nz_elev / 1000, add = TRUE)
+terra::plot(nz_elev / 1000, add = TRUE)
```
Comment (Robinlovelace) [inline 09-mapping.Rmd:938]: :+1:

**Review by Robinlovelace — COMMENTED** [review]
Looks good. Will wait to see what Jakub says and how it relates to work in progress...

**Comment by Lvulis** [comment]
That next commit for Chapter 11, I noticed that the `code/11-x.R` scripts are referred to as `10-x.R` and fixed it. Should not have pushed it in with this PR, meant to hold on to it for a separate issue so things are easier to track. A bit of a GH novice for best way to proceed - can I revert the commit or just leave it as part of this PR?

**Review by Robinlovelace — APPROVED** [review]

**Review by Robinlovelace — APPROVED** [review]

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Out of date so closing.

**Pull request closed without merging**